src/components/paperManager.js

```diff
@@ -0,0 +1,67 @@
+import { dia } from 'jointjs';
+import { highlightPadding } from '@/mixins/crownConfig';
+
+export default class {
+  constructor(paper) {
+    this._paper = paper;
```

Comment:
I'd be interested in doing that. The reason for naming this `_paper` though was to avoid infinite recursion in the getter called paper. Would using the private class field take care of that issue?

Reply:
It seems like it would, since the private property would be accessed and assigned using `this.#paper` and not just `this.paper`.

Reply:
Let's give it a spin and see what happens. :)